dm/worker/subtask.go

```diff
@@ -612,10 +614,24 @@ func (st *SubTask) unitTransWaitCondition(subTaskCtx context.Context) error {
 			if err != nil {
 				return terror.WithClass(err, terror.ClassDMWorker)
 			}
+			if st.cfg.EnableGTID {
+				gset1, err := gtid.ParserGTID(st.cfg.Flavor, loadStatus.MetaBinlogGTID)
+				if err != nil {
```

Comment:
`if err != nil {`, we still do `rc, ok := binlog.CompareGTID(gset1, gset2)`, is there any problem?